onnx/defs/tensor/defs.cc

```diff
@@ -1564,7 +1564,10 @@ ONNX_OPERATOR_SET_SCHEMA(
             "data",
             "Tensors with at least max(dims) dimensions.",
             "T",
-            OpSchema::Single)
+            OpSchema::Single,
```

Comment:
I dont see the Squeeze changes reflected in the Operators and Changelog files

Reply:
Please see line 20508 in Operators.md.